BrainPortal/app/views/sessions/new.html.erb

```diff
@@ -60,9 +60,14 @@
 
     <tr>
       <td></td>
-      <td class="field"><%= submit_tag 'Sign in', tabindex: 3 %></td>
+      <td class="field">
```

Comment:
Maybe put the register option further down, under a banner that says "Not a user? Ask for an account" or something